libs/libRocket/Source/Core/precompiled.cpp

```diff
@@ -27,3 +27,6 @@
 
 #include "precompiled.h"
 
+// XCode's ranlib complain on empty object, adding any function prevents this
+void RocketDummyFunction() {
```

Comment:
Out of curiosity, do we keep librocket up to date manually or do we use git subtrees?